addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -48,39 +47,40 @@ This configuration parameter only controls the Onkyo AVR auto-discovery process,
 
 The binding has the following configuration options, which can be set for "binding:onkyo":
 
-| Parameter | Name    | Description  | Required |
-|-----------------|------------------------|--------------|------------ |
-| callbackUrl | Callback URL | URL to use for playing notification sounds, e.g. http://192.168.0.2:8080 | no |
+| Parameter   | Name         | Description                                                              | Required |
+|-------------|--------------|--------------------------------------------------------------------------|----------|
+| callbackUrl | Callback URL | URL to use for playing notification sounds, e.g. http://192.168.0.2:8080 | no       |
 
 ## Thing Configuration
 
 The Onkyo AVR thing requires the ip address and the port to access it on.
+In the code avr1 refers to the user defined unique id of your Onkyo device. A second device could be called avr2.
```

Comment:
Consider `avr1`

Also to not falsely indicate any underlying naming scheme, how about the second example name "avr-livingroom" or sth similar...